ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/baseActivity.component.ts

```diff
@@ -23,6 +23,9 @@ import {
 import { SubmissionManager } from '../../services/serviceAgents/submissionManager.service';
 import { ConfigurationService } from '../../services/configuration.service';
 
+@Component({
+    template: ``
+})
```

Comment:
Angular 10 requires decorator if we want to use component features e.g. `@Input()`